main.nf

```diff
@@ -669,7 +669,8 @@ if(params.aligner == 'hisat2'){
     }
 }
 
-
+bed12
+.into { bed_rseqc; bed_genebody_coverage } 
```

Comment:
It'd be nicer to put this up [at the top](https://github.com/SciLifeLab/NGI-RNAseq/blob/master/main.nf#L197-L201) along with the other channel declarations I think..

Reply:
Well then it won't work if it's been generated from fastq files here: https://github.com/SciLifeLab/NGI-RNAseq/blob/master/main.nf#L439

Reply:
No, you'd need to update that process too, to generate these two channel names. But that's what we do in several other places already.